point_cloud_topic: kitti/velo/pointcloud
imu_topic: kitti/oxts/imu
gnss_topic: kitti/oxts/gps/fix

OXTS_RT3003:
  ACC_BIAS: 4.905e-5 # [m/s^2]
  ACC_STD: 0.100005 # [m/s^2]
  GYRO_BIAS: 0.01 # [deg/s]
  GYRO_STD: 0.100012 # [deg/s]

VLP16:
  HRES: 0.4 # [deg]
  VRES: 2.5 # [deg]
  VFOV_MIN: -15.0 # [deg]
  VFOV_MAX: 15.0 # [deg]
  VFOV: 30.0 # [deg]
  RANGE: 100.0 # [m]
  YFUDGE: 3 
  ZFUDGE: 1

HDL64E:
  HRES: 0.2 # [deg]
  VRES: 0.4 # [deg]
  VFOV_MIN: -24.9 # [deg]
  VFOV_MAX: 2.0 # [deg]
  VFOV: 26.9 # [deg]
  RANGE: 120.0 # [m]
  YFUDGE: 5